arch/arm64/boot/dts/qcom/sdm625-xiaomi-tissot.dts

```diff
@@ -0,0 +1,318 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2022 The Linux Foundation. All rights reserved.
+ */
+/dts-v1/;
+
+#include "msm8953.dtsi"
+#include "pm8953.dtsi"
+#include "pmi8950.dtsi"
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Xiaomi Mi A1";
+	compatible = "xiaomi,tissot", "qcom,sdm625";
+	chassis-type = "handset";
+	qcom,msm-id = <293 0>;
+	qcom,board-id = <0x1000b 0x00>;
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&gpio_key_default>, <&gpio_hall_sensor_default>;
+
+		event-hall-sensor {
+			label = "Hall Effect Sensor";
+			gpios = <&tlmm 44 GPIO_ACTIVE_LOW>;
+			linux,input-type = <EV_SW>;
+			linux,code = <SW_LID>;
+			linux,can-disable;
+		};
+
+		key-volume-up {
+			label = "Volume Up";
+			gpios = <&tlmm 85 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_VOLUMEUP>;
+		};
+	};
+
+	// FIXME clean up
+	reserved-memory {
+		/delete-node/memory@85b00000;
+
+		memory@84a00000 {
+			reg = <0x0 0x84a00000 0x0 0x1900000>;
+			no-map;
+		};
+
+		adsp_fw_mem: memory@8d600000 {
+			reg = <0x0 0x8d600000 0x0 0x1200000>;
+			no-map;
+		};
+
+		/delete-node/memory@8e700000;
+		wcnss_fw_mem: memory@8e800000 {
+			reg = <0x0 0x8e800000 0x0 0x700000>;
+			no-map;
+		};
+
+		ramoops@9ff00000 {
+			compatible = "ramoops";
+			reg = <0x0 0x9ff00000 0x0 0x00100000>;
+			record-size = <0x1000>;
+			console-size = <0x80000>;
+			ftrace-size = <0x1000>;
+			pmsg-size = <0x8000>;
+		};
+	};
+
+	vph_pwr: vph-pwr-regulator {
+		compatible = "regulator-fixed";
+		regulator-name = "vph_pwr";
+		regulator-always-on;
+		regulator-boot-on;
+	};
+};
+
+&hsusb_phy {
+	// FIXME missing supplies
+	status = "okay";
+};
+
+&i2c_2 {
+	status = "okay";
+
+	max98927_codec: audio-codec@3a {
+		compatible = "maxim,max98927";
+		reg = <0x3a>;
+
+		reset-gpios = <&tlmm 86 GPIO_ACTIVE_LOW>;
+
+		vmon-slot-no = <1>;
+		imon-slot-no = <1>;
+
+		#sound-dai-cells = <1>;
+	};
+
+	led-controller@45 {
+		compatible = "awinic,aw2013";
+		reg = <0x45>;
+
+		vcc-supply = <&pm8953_l10>;
+
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		led@0 {
+			reg = <0>;
+			led-max-microamp = <5000>;
+			function = LED_FUNCTION_INDICATOR;
+			color = <LED_COLOR_ID_WHITE>;
+		};
+	};
+};
+
+&i2c_3 {
+	status = "okay";
+
+	// FIXME compatible
+	touchscreen@38 {
+		compatible = "edt,edt-ft5x06";
```

Comment:
@JIaxyga We need to use a proper compatible for this, dmesg should be good enough for identification which model this is

Reply:
For tissot, mido and daisy according to downstream, focaltech,5435 is used. But the closest one in the mainline that works is edt, edt-ft5406. Let's use this

Reply:
Do you have those devices to confirm that they're actually those models? Mainline driver should print the number here https://github.com/torvalds/linux/blob/master/drivers/input/touchscreen/edt-ft5x06.c#L1293-L1295 (to show easiest use `#define DEBUG` at the very top of the file before any include, or change to dev_err) or in input device name (check evtest) or check sysfs `model` attribute. 

I'd try creating `focaltech,ft5446` and `focaltech,ft5435` compatibles in dt bindings and driver then. Not 100% if maintainer wants this naming but it sounds most sensible based on my info.

Also there's a [somewhat interesting pdf I found](https://forum-raspberrypi.de/attachment/18931-tp-driver-porting-guide-for-qualcomm-v2-0-20170919-pdf/).

Reply:
I only have tissot.  I'll try to find out the model today through debug

Reply:
@JIaxyga any update on this one?

Reply:
> @JIaxyga any update on this one?

My graphic card in my computer is broken and I can't check yet.  But I'll have to check in the next few days

Reply:
I am very sorry for the delay. I got it:
[    0.711016] edt_ft5x06 2-0038: Model "generic ft5x06 (d0)", Rev. "	", 1024x1024 sensors